graph/src/runtime/mod.rs

```diff
@@ -240,6 +240,21 @@ pub enum IndexForAscTypeId {
     NearChunkHeader = 84,
     NearBlock = 85,
     NearReceiptWithOutcome = 86,
+    SolanaBlock = 87,
```

Comment:
Need to be aligned with `graph-ts` code base. I think we should start to have dedicated "range" per chain. Solana could be 200 - 399, Tendermint 400 - 699 (etc.) to ease future addition. Maybe there is restrictions of doing that.

Reply:
Agree we should use ranges